rclpy/rclpy/subscription.py

```diff
@@ -33,6 +53,7 @@ def __init__(
          callback: Callable,
          callback_group: CallbackGroup,
          qos_profile: QoSProfile,
+         event_callbacks: SubscriptionEventCallbacks,
```

Comment:
Curious why we're using a positional argument rather than optional?

Reply:
This is an implementation API, the user-facing interface is `Node.create_subscription()`, which uses optional arguments